node/src/chain_spec/oak.rs

```diff
@@ -21,7 +21,8 @@ const TOKEN_SYMBOL: &str = "OAK";
 const SS_58_FORMAT: u32 = 51;
 static RELAY_CHAIN: &str = "rococo-local";
 static STAGING_RELAY_CHAIN: &str = "rococo";
-const REGISTERED_PARA_ID: u32 = 2114;
+static LIVE_RELAY_CHAIN: &str = "polkadot";
```

Comment:
we confirmed this from their chainspec, right?

Reply:
believe so! https://raw.githubusercontent.com/paritytech/polkadot/master/node/service/chain-specs/polkadot.json

Reply:
hmmm actually now that i look at the other chain specs in that file (kusama and rococo), our names don't quite match the names on their chain spec